app/jobs/assignment_repo/create_github_repository_job.rb

```diff
@@ -13,150 +13,37 @@ class CreateGitHubRepositoryJob < ApplicationJob
     # user       - The User that the AssignmentRepo will belong to
     # retries    - The number of times the job will automatically retry
     #
-    # rubocop:disable MethodLength
-    # rubocop:disable AbcSize
     def perform(assignment, user, retries: 0)
```

Comment:
This method is so clean now 😍 

Generally, keeping logic out of our jobs is a great thing. Brought out into a model or class of it's own, the logic can be reused elsewhere. ✨